daemon/exercise.go

```diff
@@ -45,11 +47,16 @@ func (d *daemon) ListCategories(ctx context.Context, req *pb.Empty) (*pb.ListCat
 
 	for _, c := range cats {
 		// Render markdown from orgdescription to html
-		extensions := parser.CommonExtensions | parser.AutoHeadingIDs | parser.HardLineBreak
-		parser := parser.NewWithExtensions(extensions)
 
 		md := []byte(c.CatDescription)
-		unsafeHtml := markdown.ToHTML(md, parser, nil)
+		var buf bytes.Buffer
+		renderer := goldmark.New(
+			goldmark.WithRendererOptions(html.WithUnsafe()),
+		)
+		if err := renderer.Convert(md, &buf); err != nil {
+			log.Error().Msgf("Error converting to commonmark: %s", err)
+		}
+		unsafeHtml := buf.Bytes()
```

Comment:
let's make this part as function and just call it.

Reply:
Haha im so stupid, i'll do that as soon i get time! :D